src/main/java/org/altbeacon/beacon/distance/ModelSpecificDistanceCalculator.java

```diff
@@ -98,6 +99,19 @@ public double calculateDistance(int txPower, double rssi) {
         return mDistanceCalculator.calculateDistance(txPower, rssi);
     }
 
+    DistanceCalculator _findCalculatorForModelTest(AndroidModel model) {
```

Comment:
If you want to expose this method for testing, I would prefer to remove the private modifier from the main method (leaving it with no modifier or changing to protected) rather than making a new wrapper method for testing purposes.  This has been the general approach elsewhere in the library.